css-flexbox/align-items.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```

Comment:
The flex-basis prop should be on the `.card-wrapper` class

Reply:
`#card-deck` should still have `display: flex` as a property since that is the container element. 
What I meant to say was `flex-basis: calc(100% / 3)` should be a property of `.card-wrapper` - `flex-basis` is a property that you apply to flex items, not the flex container.